batch/Makefile

```diff
@@ -1,9 +1,26 @@
 .PHONY: build build-test \
   push push-test \
   run-docker run \
-  test test-local deploy
+  test test-local deploy clean
 
 PROJECT = $(shell gcloud config get-value project)
+PY_FILES = $(shell find batch -iname \*.py -not -exec git check-ignore -q {} \; -print)
+PY_CHECKERS = build/flake8 build/pylint
+
+clean:
+	-rm -r build
```

Comment:
should it be `-rm` or `rm`?

Reply:
Without `-r`, my `rm` complains:
```
rm: build: is a directory
```

Reply:
Should have been more clear. Do you need the dash in front of `rm`?

Reply:
Can you respond to this and then I'll approve?

Reply:
Yes, `-` means if this command fails, do not fail the build. It's OK if the `build` directory already does not exist.